src/components/Header.spec.js

```diff
@@ -0,0 +1,11 @@
+
+    import {render, screen} from '@testing-library/react'
+    import Header from './Header'
+      
+    describe('Header', () => {
+        it('renders', () => {
+            render(<Header />)
+            expect(screen.getByText('Header')).toBeInTheDocument()
+        })
+    })
```

Comment:
It looks like you forgot this test? 😉

Reply:
Looks like it! You should test if `pageName` will be displayed when passing the prop.

Reply:
I added the test for the pageName and it works :)